tidb.go

```diff
@@ -88,7 +89,7 @@ var (
 	}
 	stores = make(map[string]kv.Driver)
 	// Debug is the switch for pprof. If Debug is true, tidb will start with pprof on.
-	Debug = true
+	Debug = os.Getenv("TIDB_DEBUG") == "1"
```

Comment:
Right now, TIDB_PPROF should be 1 by default.
We may set to 0 when we release beta version.